style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#grid-container {
+  display: grid;
+  grid-auto-flow: column;
+  margin-right: 1rem;
+}
+
+.cell {
+  border: 1px solid black;
+  width: 30px;
+  height: 30px;
+}
+
+.cell-painted {
+  background-color: black;
+  border: 1px solid black;
+  width: 30px;
+  height: 30px;
+}
```

Comment:
Don't repeat the same styles in multiple classes.
You can keep the main `cell` styles, and leave only `background-color` in `cell-painted`.
```html
<div class="cell" /> // this will be your normal cell
<div class="cell cell-painted" /> // this will be your painted cell
```